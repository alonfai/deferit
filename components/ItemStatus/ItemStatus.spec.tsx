// import { render, fireEvent } from '@testing-library/react-native';
// import ItemSummary from './ItemSummary';

describe('<ItemStatus />', () => {
  it('demo', () => {
    expect(1).toEqual(1);
  });
});
