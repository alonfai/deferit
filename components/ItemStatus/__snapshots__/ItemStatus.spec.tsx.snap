// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ItemStatus /> Render successfully 1`] = `
<View
  accessibilityLabel="Status Label"
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text />
</View>
`;
